docs/content/en/docs/getting-started/_index.md

```diff
@@ -15,58 +25,266 @@ In your local environment the following tools are required for provisioning a K8
+
+Depending upon your K8 configuration, initialization of your K8ssandra installation can take a few minutes. To check the status of your K8ssandra deployment, use the `kubectl` command:
+
+```bash
+kubectl get pods
+NAMESPACE     NAME                                                  READY   STATUS      RESTARTS   AGE
+default       k8ssandra-cass-operator-6666588dc5-rrbwx              1/1     Running     0          15m
+default       k8ssandra-dc1-default-sts-0                           2/2     Running     0          15m
+default       k8ssandra-dc1-stargate-7db4dbfdd5-dxk5v               1/1     Running     0          15m
+default       k8ssandra-grafana-b6f7978c4-cwdwg                     2/2     Running     0          15m
+default       k8ssandra-kube-prometheus-operator-5556885bd6-2tn4t   1/1     Running     0          15m
+default       k8ssandra-reaper-k8ssandra-64cc9d57d9-rc2z4           1/1     Running     0          12m
+default       k8ssandra-reaper-k8ssandra-schema-pxdcz               0/1     Completed   0          12m
+default       k8ssandra-reaper-operator-cc46fd5f4-hlmck             1/1     Running     0          15m
+default       prometheus-k8ssandra-kube-prometheus-prometheus-0     2/2     Running     1          15m
+```
 
-In later steps, you can upgrade your k8ssandra via `helm upgrade` commands, for example to access services from outside Kubernetes via a Traefik Ingress controller.
+NOTE: k8ssandra prefixes the pod names. Stargate takes 4 minutes to register as ready.
 
-## Defaults
 
-K8ssandra comes out of the box with a set of default values tailored to getting up and running quickly.  These defaults
-are intended to be a great starting point for smaller-scale local development and should be carefully customized for 
-production deployments.
 
-For more information on the configuration options available, and the developer focused defaults provided, take a look 
-at the [k8ssandra reference]({{< ref "/docs/reference/k8ssandra/" >}}) documentation.
+
+Once all the pods are in the `Running` or `Completed` state, you can check the health of your K8ssandra cluster.
+
+{{% alert title="Tip" color="success" %}}
+The pod `k8ssandra-reaper-k8ssandra-schema-xxxxx` runs once and does not persist.
+{{% /alert %}}
+
+The actual Cassandra node name from the listing above is `k8ssandra-dc1-default-sts-0`. If you've configured multiple nodes, `sts-0` will increment.
+
+We'll use node name `k8ssandra-dc1-default-sts-0` throughout the following sections.
+
+To check the health of your K8ssandra cluster:
+
+1. Confirm that the Cassandra operator is `Ready`:
+
+    ```bash
+    kubectl describe CassandraDataCenter dc1 | grep "Cassandra Operator Progress:"
+       Cassandra Operator Progress:  Ready
+    ```
+
+1. Get K8ssandra superuser credentials:
+
+    * K8ssandra superuser name:
+
+        ```bash
+        kubectl get secret k8ssandra-superuser -o jsonpath="{.data.username}" | base64 --decode | more
```

Comment:
I'm not sure we want to include `more` here. The purpose being that there is a highlighted `%` in your shell. This is a [behavior of zshell and specific to your environment](https://unix.stackexchange.com/questions/167582/why-zsh-ends-a-line-with-a-highlighted-percent-symbol). The `more` and `(END)` blocks are a bit distracting in my opinion.

Reply:
Consider taking this value and placing it in an environment variable that you can then reference throughout the rest of this guide.

```suggestion
        K8SSANDRA_USERNAME=`kubectl get secret k8ssandra-superuser -o jsonpath="{.data.username}" | base64 --decode`
        echo "K8ssandra Username: $K8SSANDRA_USERNAME"
        K8ssandra Username: ZGVtby1zdXBlcnVzZXI=
```

(Note the above has not been tested)